test/e2e/tests/test_cross_region.py

```diff
@@ -0,0 +1,105 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+"""Integration test for ACKs Cross Region Support.
+"""
+
+import logging
+from acktest.aws.identity import get_region
+import boto3
+import botocore
+import pytest
+import time
+from typing import Dict
+
+from acktest.resources import random_suffix_name
+from acktest.k8s import resource as k8s
+from e2e import (
+    service_marker,
+    create_sagemaker_resource,
+    assert_tags_in_sync,
+)
+from e2e.replacement_values import REPLACEMENT_VALUES, XGBOOST_V1_IMAGE_URIS
+from e2e.common import config as cfg
+
+
+def get_cross_region():
+    region = get_region()
+    if region != "us-west-2":
+        region = "us-west-2"
+    else:
+        region = "us-east-1"
+    return region
```

Comment:
Maybe it simpler to read if you just do
```python
def get_cross_region():
    region = get_region()
    if region == "us-west-2":
        return "us-west-2"
    return "us-east-1"
```

Reply:
yeah this looks cleaner, is a != if I'm correct however.